EESSI-install-software.sh

```diff
@@ -229,6 +229,27 @@ if [[ "${EESSI_CVMFS_REPO}" != /cvmfs/dev.eessi.io ]]; then
     ${TOPDIR}/install_scripts.sh --prefix ${EESSI_PREFIX}
 fi
 
+echo ">> Configuring EasyBuild..."
+
+# Make sure that we use the EESSI_CVMFS_INSTALL
+# Since the path is set when loading EESSI-extend, we reload it to make sure it works - even if it is already loaded
+# Note we need to do this after running install_cuda_and_libraries, since that does installations in the EESSI_SITE_INSTALL 
```

Comment:
@trz42 This comment is not valid any more, but I think it doesn't matter in general as the `install_cuda_and_libraries.sh` is _not_ sourced but called as a script?

Reply:
So doesn't actually affect the installation path